plugins/ote-migration/README.md

```
@@ -0,0 +1,454 @@
+## Installation
+
+This plugin is available through the ai-helpers marketplace:
+
+```bash
+/plugin marketplace add openshift-eng/ai-helpers
+/plugin install ote-migration@ai-helpers
+```
+
+## Usage
+
+**IMPORTANT: Identify Your Sig Tags**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)